npm/vue2/rollup.config.mjs

```diff
@@ -3,6 +3,9 @@ import json from '@rollup/plugin-json'
 import replace from '@rollup/plugin-replace'
 
 const config = {
+  dtsOptions: {
+    respectExternal: false
```

Comment:
was the issue that were were rolling up the vue `.d.ts` files from `2.6.12` and that was causing issues?

Reply:
No, the issue is not related to rolling up the Vue .d.ts files from 2.6.12. This problem occurs in versions[ "2.7+"](https://github.com/cypress-io/cypress/pull/28818/commits/89a83cbc8399627c894fc948e2eb7195e9f06108) instead.

Reply:
This won't affect Vue 2. In most cases, this option (respectExternal) doesn't need to be enabled, at least not for Vue as Vue itself already includes its own types. The documentation of rollup-plugin-dts also specifies that this option is disabled by default.

The main issue is that after upgrading to Vue 2.7+, the respectExternal option of rollup-plugin-dts was causing the npm/vue2 build task to fail. That's why I disabled this option for npm/vue2.